component/FloatingAction.js

```diff
@@ -105,29 +129,42 @@ class FloatingAction extends Component {
       }]
     };
 
+    if (this.props.inanimate || (this.props.compact && this.props.actions.length === 1 && this.props.inanimateCompact)) {
+      animatedViewStyle = {};
+    }
+
     const Touchable = getTouchableComponent();
 
+    let bgColor = { backgroundColor: '#1253bc' };
+    if (buttonColor) {
+      bgColor = { backgroundColor: buttonColor };
+    }
+
     return (
       <Animated.View
-        style={[styles.buttonContainer, styles[`${position}Button`], { backgroundColor: buttonColor }, animatedVisibleView]}
+        style={[styles.buttonContainer, styles[`${position}Button`], { backgroundColor: buttonColor || '#1253bc' }, animatedVisibleView]}
```

Comment:
the **backgroundColor** was defined inside `FloatingAction.defaultProps` so is not necessary to add it here.

Reply:
In my case it became necessary, when I pass null it goes white.